.github/actions/setup-environment/action.yml

```diff
@@ -241,6 +241,7 @@ runs:
 
         if [ "$ENVIRONMENT_NAME" = "Branch" ]; then
           PR_NUMBER=${{ github.event.number || github.event.inputs.pr-number || inputs.pr-number }}
+          IS_CONTENTFUL=${{ contains(github.event.pull_request.labels.*.name, 'contentful') }}
```

Comment:
maybe HAS_CONTENTFUL_LABEL? only IS_CONTENTFUL sounds a little bit weird

Reply:
agreed, we have already too many of 'is_contentful's in the codebase

Reply:
Done.